test/DynamicDependency/data/WindowsAppRuntime.Test.Singleton.Msix/WindowsAppRuntime.Test.Singleton.Msix.vcxproj

```diff
@@ -107,6 +124,17 @@
   <ItemGroup>
     <Xml Include="appxmanifest.xml" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\dev\PushNotifications\PushNotificationsLongRunningTask.ProxyStub\PushNotificationsLongRunningTask.ProxyStub.vcxproj">
```

Comment:
BTW you can replace `..\..\..\..\dev\` with `$(RepoRoot)\dev\` then you're not path relative. 

It's a best-practice not to use .. relative path'ing, but we haven't done anything large scale push to clean up the repo (yet). More of an opportunistic change.

NOTE: RepoRoot has no trailing backslash (unlike other paths in VS e.g. $(OutDir) which always end in backslash)